src/main/java/com/kodilla/ecommercee/domain/OrderEntity.java

```diff
@@ -0,0 +1,26 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity(name = "ProductOrder")
+public class OrderEntity {
```

Comment:
klasa nazywa się OrderEntity... czy jesteś pewny że w tym pakiecie nie będzie innej klasy związanej z Order? linijkę wyżej nadałeś ręcznie nazwę w bazie jako "ProductOrder" - czy to jest celowo?

Reply:
W zasadzie to jest to podobna sytuacja, z którą spotkałem się poprzednio. Kiedy name = "Order", to nie chce się skompilować, bo Order jest deskryptorem. Stąd nadałem ją początkowo "ProductOrder", ale równo dobrze można nic nie podawać, to wtedy nazwa będzie taka sama jak nazwa klasy.